Examples/Algorithms/Digitization/src/DigitizationAlgorithm.cpp

```diff
@@ -304,7 +304,7 @@ ActsExamples::DigitizationAlgorithm::localParameters(
   for (const auto& ch : channels) {
     auto bin = ch.bin;
     Acts::ActsScalar charge =
-        geoCfg.digital ? 1. : geoCfg.charge(ch.activation, ch.activation, rng);
+        geoCfg.digital ? 1. : geoCfg.charge(ch.activation, rng);
```

Comment:
Oh so this was just duplicated? Or is this maybe a mistake at the call-site here?

Reply:
I was not sure about that. but as far as I could see this params was never used so I dropped it

Reply:
I recently discussed this with @asalzburger, but unfortunately don't remember the meaning of this right now. However, since it is not used at the moment, I think we can remove it and eventually introduce it again if needed.